Fixed location bias for Nomnatim geocoding

Kai15116 opened the pull request:
Modified nominatim geocoder wrapper so it looks for location in Edmonton automatically. 
I also modified main files so it configures the domain specific files automatically.

ssanner commented:
> Modified nominatim geocoder wrapper so it looks for location in Edmonton
automatically.

I understand this change is in a domain specific class, but it still seems
to me that specific restrictions like "Edmonton" are better specified in
the configuration files rather than hardcoded in Python.  If I change the
data to Toronto, I have to know to change this part of the code as well.
-Scott

On Wed, Aug 2, 2023 at 12:10 PM Kai15116 ***@***.***> wrote:

> Modified nominatim geocoder wrapper so it looks for location in Edmonton
> automatically.
> I also modified main files so it configures the domain specific files
> automatically.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/D3Mlab/llm-convrec/pull/35
> Commit Summary
>
>    - 642aa68
>    <https://github.com/D3Mlab/llm-convrec/pull/35/commits/642aa68db4acd9bf24e9c87a8f4a793573021b25>
>    Fixed location bias and modified main files so it configures the domain
>    specific configs automatically
>
> File Changes
>
> (3 files <https://github.com/D3Mlab/llm-convrec/pull/35/files>)
>
>    - *M* clothing_main.py
>    <https://github.com/D3Mlab/llm-convrec/pull/35/files#diff-b17f61bc6b1e4954928174dfa1d5a597120e6ca92a0b8585926ebf24133a21f7>
>    (2)
>    - *M*
>    domain_specific/classes/restaurants/geocoding/nominatim_wrapper.py
>    <https://github.com/D3Mlab/llm-convrec/pull/35/files#diff-4fa23ace15d4304f3899ad1cf89d7d06416b06ec985f357b3b77919ce4bb8259>
>    (7)
>    - *M* restaurant_main.py
>    <https://github.com/D3Mlab/llm-convrec/pull/35/files#diff-c3c7d35acaa5244e58389b4fcafd6e5bf6c7452dbed090bbe38d837313c57fec>
>    (4)
>
> Patch Links:
>
>    - https://github.com/D3Mlab/llm-convrec/pull/35.patch
>    - https://github.com/D3Mlab/llm-convrec/pull/35.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/D3Mlab/llm-convrec/pull/35>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABRA3JAEVTLYIOPOSWJPCRLXTJ3XZANCNFSM6AAAAAA3BQFW5Q>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

Kai15116 commented:
> Modified nominatim geocoder wrapper so it looks for location in Edmonton
> automatically. I understand this change is in a domain specific class, but it still seems to me that specific restrictions like "Edmonton" are better specified in the configuration files rather than hardcoded in Python. If I change the data to Toronto, I have to know to change this part of the code as well. -Scott
> […](#)
> On Wed, Aug 2, 2023 at 12:10 PM Kai15116 ***@***.***> wrote: Modified nominatim geocoder wrapper so it looks for location in Edmonton automatically. I also modified main files so it configures the domain specific files automatically. ------------------------------ You can view, comment on, or merge this pull request online at: #35 Commit Summary - [642aa68](https://github.com/D3Mlab/llm-convrec/commit/642aa68db4acd9bf24e9c87a8f4a793573021b25) <[642aa68](https://github.com/D3Mlab/llm-convrec/pull/35/commits/642aa68db4acd9bf24e9c87a8f4a793573021b25)> Fixed location bias and modified main files so it configures the domain specific configs automatically File Changes (3 files <https://github.com/D3Mlab/llm-convrec/pull/35/files>) - *M* clothing_main.py <https://github.com/D3Mlab/llm-convrec/pull/35/files#diff-b17f61bc6b1e4954928174dfa1d5a597120e6ca92a0b8585926ebf24133a21f7> (2) - *M* domain_specific/classes/restaurants/geocoding/nominatim_wrapper.py <https://github.com/D3Mlab/llm-convrec/pull/35/files#diff-4fa23ace15d4304f3899ad1cf89d7d06416b06ec985f357b3b77919ce4bb8259> (7) - *M* restaurant_main.py <https://github.com/D3Mlab/llm-convrec/pull/35/files#diff-c3c7d35acaa5244e58389b4fcafd6e5bf6c7452dbed090bbe38d837313c57fec> (4) Patch Links: - https://github.com/D3Mlab/llm-convrec/pull/35.patch - https://github.com/D3Mlab/llm-convrec/pull/35.diff — Reply to this email directly, view it on GitHub <#35>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABRA3JAEVTLYIOPOSWJPCRLXTJ3XZANCNFSM6AAAAAA3BQFW5Q> . You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

That is true, I moved the location bias to config file.

kathy23980 reviewed (approved):
looks good!
